Input/input_sup2.xml

```diff
@@ -23,6 +23,11 @@
       <Parameter name="Delta" type="double" value="15.0"/>
       <Parameter name="Tolerance" type="double" value="0.01"/>
     </ParameterList>
+  </ParameterList>
 
+  <ParameterList name="result_description">
+    <Parameter name="ExpectedPressure" type="double" value="0.0060690998"/>
+    <Parameter name="ExpectedPressureTolerance" type="double" value="1e-6"/>
```

Comment:
The tolerance is quite high compared to the value, can we set this to a smaller value and the tests still pass?

Reply:
I use the current output in the `ExpectedPressure`. I can set it by printing the current output using `std::setprecision(16)`. And then I can use a lower tolerance. I am not sure what is a good value for this tolerance. Should I use `1e-10`? The test still passes if I use this tolerance. I also intentionally broke the code by using an incorrect value for the expected result and it works.